ci-operator/jobs/terraform-redhat/terraform-provider-red-hat-cloud-services/terraform-redhat-terraform-provider-red-hat-cloud-services-main-periodics.yaml

```diff
@@ -1,31 +1,21 @@
 periodics:
 - agent: kubernetes
   cluster: build05
-  cron: 12 22 * * *
+  cron: 11 3 * * *
   decorate: true
   decoration_config:
     skip_cloning: true
   extra_refs:
   - base_ref: main
     org: terraform-redhat
-    repo: terraform-provider-ocm
+    repo: terraform-provider-red-hat-cloud-services
   labels:
     ci-operator.openshift.io/cloud: aws
     ci-operator.openshift.io/cloud-cluster-profile: aws-qe
     ci-operator.openshift.io/cluster: build05
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-terraform-redhat-terraform-provider-ocm-main-managed-oidc-config-e2e-periodic
```

Comment:
can you bring this back? otherwise we won't be notified for periodic job failures

Reply:
alternatively, there's a way to configure it somewhere else where ``make update`` won't remove it on job renames
an example: https://github.com/openshift/release/pull/34274

Reply:
Can we use the new job, and have it reporting about failures?
I would like to be consistent with the names

Reply:
yes, you can have any job running periodically and have it reporting
my comment is about removing the reporting part, which is part of what happened in this PR